src/lib/state/transactions.ts

```diff
@@ -37,7 +38,14 @@ export interface OutputSwapTransactionInfo extends SwapTransactionInfo {
   maximumInputCurrencyAmount: string
 }
 
-export type TransactionInfo = ApprovalTransactionInfo | SwapTransactionInfo
+export interface WrapTransactionInfo extends BaseTransactionInfo {
+  type: TransactionType.WRAP
+  unwrapped: boolean
```

Comment:
Is `unwrapped` copied from the app? I'd rather have this be `wrapType`, with the options being `WRAP | UNWRAP`.

Reply:
yeah, this is copied over like this for consistency.